docs/manage_users.md

```diff
@@ -0,0 +1,129 @@
+# Manage users in Percona Everest
+
+Percona Everest provides user management capabilities, enabling you to securely log in through either the Percona Everest UI or the API.
+
+Local user management involves administering Percona Everest users to ensure secure access to database resources. This encompasses tasks such as creating and deleting users, updating their passwords, etc.
+
+When you install Percona Everest, an `admin` user is automatically created, granting full access to the system.
+
+!!! note alert alert-primary "Note"
+
+    It is recommended that you use the admin account for the initial configuration and subsequently transition to Single Sign-On (SSO) integration.
+
+
+## Overview
+
+All user accounts are stored in the YAML format within the `everest-accounts` Secret in the `everest-system` namespace:
+
+??? example "User accounts: YAML format"
+        
+    ```{.text .no-copy}
+    apiVersion: v1
+    kind: Secret
+    metadata:
+        name: everest-accounts
+        namespace: everest-system
+    data:                    
+      users.yaml: YWRtaW46CiAgcGFzc3dvcmRIYXNoOiBhZG1pbgogIGVuYWJsZWQ6IHRydWUKICBjYXBhYmlsaXRpZXM6CiAgICAtIGxvZ2lu
+    ```
+
+Decoding the base64 encoded value will yield a YAML that appears as follows:
+
+
+??? example "YAML after decoding"
+
+    ```{.text .no-copy}
+    admin:
+        passwordHash: <password>
+    enabled: true
+    capabilities:
+        - login
+    ```
+
+## User management commands
+
+This section provides a list of CLI commands for managing users in Percona Everest.
+
+### Retrieve password
+
+You can retrieve the password for the `admin` user by running the following command:
+
+```sh
+everestctl accounts initial-admin-password
+```
+
+!!! note alert alert-primary "Note"
+
+    The passwords in this Secret are stored as hashes. However, the initial admin user has the password stored as plain text for convenient retrieval later on. We strongly recommend that you update the admin password after installation.
+
+
+
+### Update the password
+
+To update the password for an existing user:
+
+```sh
+everestctl accounts set-password -u <username> --new-password $PASSWORD
+```
+
+### Create a new user
+
+To create a new user:
+
+```sh
+everestctl accounts create -u <username> -p $PASSWORD
+```
+
+This user will have all the permissions to the system.
```

Comment:
Please note that this is true only until RBAC is added. After that, new users will have no permissions, and the Everest admin will have to explicitly add new permissions for new users.

Reply:
@mayankshah1607 In that case should we add a note?

Reply:
Yes, that would be nice. We can replace this line with a note that says:

```
As of v1.0.0, newly added users have full access to the system. Once RBAC support is added, Everest admins will be able to manage permissions for users, granting them fine-grained control over database resources.
```

(And once RBAC is in, we should also remember to remove/update this note)

Reply:
ok. Sure.

Reply:
Fixed.